loki/transformations/transpile/tests/test_transpile.py

```diff
@@ -19,6 +19,8 @@
 from loki.transformations.transpile import FortranCTransformation
 from loki.transformations.single_column import SCCLowLevelHoist, SCCLowLevelParametrise
 
+# pylint: disable=too-many-lines
+
```

Comment:
We should start splitting the tests into logical units (but that can be a separate PR)